splunk_add_on_ucc_framework/ucc_ui_lib/package.json

```diff
@@ -20,7 +20,12 @@
     "test:ci": "cross-env JEST_JUNIT_OUTPUT_DIR=./test-reports JEST_JUNIT_OUTPUT_NAME=unit-results.xml JEST_JUNIT_CLASSNAME=unit yarn run test --ci --reporters=default jest-junit --coverage --coverageDirectory=coverage_report/coverage_maps_unit --coverageReporters=json",
     "test:watch": "jest --watch"
   },
-  "dependencies": {},
+  "dependencies": {
+    "css-loader": "^5.2.0",
+    "node-sass": "^5.0.0",
```

Comment:
Can you verify if we can move sass dependencies to `devDependencies`?

Reply:
Yes sure, these changes will be incorporated in the next PR